README.md

```diff
@@ -297,7 +313,7 @@ in `/var/lib/bdii/gip/provider/cloud-info-provider`.
 
 ### Generate complete BDII information
 
-**This does not generate GlueSchema 1.3 information and will fail SAM tests**
+Warning: **This does not generate GLUE1.3 information and will fail SAM tests**
```

Comment:
Really? SAM ?

Reply:
We like living in the past :) 
I would completely remove this "Generate complete BDII information" section and option, as it has brought more trouble than anything in the past.

Reply:
OK, but that's not really the topic of this PR, I just rephrased to avoid a linting error...